CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+
+If you would like to have an interactive plot with `panel` widgets included, you will need to pass in an interactive dataframe (which is an hvPlot representation) into the `hvplot` function. This can be done by calling the `interactive()` method on your `pandas` dataframe as: 
+```
+df = pd.read_csv('somefile.csv')
+idf = df.interactive()
+```
+You can then call the `.hvplot()` function as you would call `.plot()` on a pandas dataframe.
+
+
+## Adding a plot (w or w/o widgets) to the dashboard
+To add the plot to the dashboard, follow the steps used in `app.py`. 
+```
+# Global list of panel objects to display on the website
+panel_objs = []
+
+# Create an object of the plot class
+plot_obj = PlotClass(...)
+
+# Create an object of the widget class
+widget = WidgetClass(...).create_widget()
+
+# Add the panel object to global list
+create_panel_object(panel_objs, "some title", widgets=[widget], plots=[plot_obj])
+```
+
+Note: If you would like to pass in custom arguments to the `plot` method, you can create the plot in `app.py` itself; before passing it into `create_panel_object` as:
+```
+try: 
+    plot_obj = PlotClass(...).plot(arg=arg)
+except Exception:
+    plot_obj = None
+create_panel_object(panel_objs, "some title", plots=[plot_obj])
+```
+
+
+## Using the Dashboard wrapper locally
+Once you have installed the package from source as per the instructions in the [`README`](https://github.com/autogluon/autogluon-dashboard/blob/main/README.md), you should have access to the `agdash` command. 
+```
+agdash --help
+usage: agdash [-h] --per_dataset_csv  --agg_dataset_csv  [--hware_metrics_csv]  [--s3_bucket ] [--s3_prefix ] [--s3_region ]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --per_dataset_csv     Location of csv file of all datasets+frameworks data to upload to S3 bucket 
+  --agg_dataset_csv     Location of csv file of aggregated data across all frameworks to upload to S3 bucket                   
+  --hware_metrics_csv   Location of csv file of hardware metrics to upload to S3 bucket
+  --s3_bucket           Name of S3 bucket to upload dashboard contents to
+  --s3_prefix           Prefix for S3 URL - subfolder
+  --s3_region           S3 Region to deploy the dashboard website. Should be the same region as s3_bucket
+```
+
+If a bucket and region is not specified, it will default to the AutoGluon bucket and region. You must have the requisite permissions to upload contents to the AutoGluon bucket. 
+<br> If you provide your own bucket and region, you will need to the following: 
+1. Make the bucket public so that the python web-app can access the csv files from it 
+2. Enable web hosting for the bucket (if you are an Amazon employee this may raise a SEV-2)
```

Comment:
Let's not mentioning the amazon employee part here. Also you'll get SEV-2 in step 1 already.

Let's instead add a caution in step 1 let the user know open the bucket to public will expose any sensitive data

Reply:
Okay will make the change